gammapy/image/tests/test_maps.py

```diff
@@ -103,3 +107,24 @@ def test_reproject(self):
         skymap_1.fill(1)
         skymap_1_repr = skymap_1.reproject(skymap_2)
         assert_allclose(skymap_1_repr.data, np.full((100, 100), 1))
+
+@requires_data('gammapy-extra')
+class TestSkyMapCrab():
+    """
+    Test sky map class.
+    """
+    def Crab_coord(self):
+        coord = SkyCoord(83.63, 22.01, unit='deg').galactic
+        return coord
+
+    def setup(self):
+        center = self.Crab_coord()
+        self.skymap  = SkyMap.empty(nxpix=250, nypix=250, binsz=0.02, xref=center.l.deg,
+                         yref=center.b.deg, proj='TAN', coordsys='GAL')
+
+    def test_center(self):
```

Comment:
@adonath 
add this test for the center by creating an image centered on the Crab position:
```
In [17]: SkyCoord.from_name("Crab").galactic
Out[17]: 
<SkyCoord (Galactic): (l, b) in deg
    (184.55745788, -5.78435671)>
```
When you use the method center of the skymap it gives:
```
<SkyCoord (Galactic): (l, b) in deg
    (184.55974014, -5.78918015)>
```
when you use the line I wrote before:
center = SkyCoord.from_pixel(self.header["NAXIS1"] / 2., self.header["NAXIS2"] / 2., exposure.wcs)
it gives:
```
<SkyCoord (Galactic): (l, b) in deg
    (184.54968906, -5.77918006)>
```